tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -98,7 +102,9 @@ suite('Git with ssh workflow', async () => {
 
         data.metadata!.name = wsNameCheckPropagatingKeys;
         await testWorkspaceUtils.createWsFromDevFile(data);
+        await driverHelper.reloadPage();
```

Comment:
`releadPage()` looks redundant in here if next command `navigateToUrl()` will change current page as well.

Reply:
In this case not. Need to update view of dashboard that next command has opened expected page. Probably, it's related with known issue: https://github.com/eclipse/che/issues/19020

Reply:
I see.
Let then reference the issue https://github.com/eclipse/che/issues/19020 in code to explain the propose of `reloadPage()` command.